cwms-servers/CWMS_HOME/bin/loggingSetup.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) 2024
+#  United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+#  All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+#  Source may not be released without written approval from HEC
+
+import json
+import logging
+import logging.config
+import os
+import sys
+
+
+def setup_logging():
+    cwms_home = os.getenv('CWMS_HOME')
+    if (cwms_home == None):
+        (logging.getLogger("logging_setup")
+         .warning("Could not load python logging config. CWMS_HOME not defined."))
+    else:
+        path = cwms_home + '/config/system_config/logging_config.json'
```

Comment:
do you want to prefix this as python logging config?

Reply:
Good idea, updated